Windows release build failing

https://github.com/ANTsX/ANTs/actions/runs/6001492622/job/16324090397#step:7:9912

```
D:\a\ANTs\ANTs\Examples\sccan.cxx(425,8): error C2039: 'random_shuffle': is not a member of 'std' [D:\a\_temp\build\ANTS-build\Examples\l_sccan.vcxproj] [D:\a\_temp\build\ANTS.vcxproj]
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\iomanip(23,1): message : see declaration of 'std' [D:\a\_temp\build\ANTS-build\Examples\l_sccan.vcxproj]
  D:\a\ANTs\ANTs\Examples\sccan.cxx(1384,24): message : see reference to function template instantiation 'vnl_matrix<double> ants::PermuteMatrix<Scalar>(vnl_matrix<double>,bool)' being compiled [D:\a\_temp\build\ANTS-build\Examples\l_sccan.vcxproj]
  D:\a\ANTs\ANTs\Examples\sccan.cxx(2380,7): message : see reference to function template instantiation 'int ants::SVD_One_View<3,double>(itk::ants::CommandLineParser *,unsigned int,unsigned int,unsigned int,unsigned int,unsigned int,unsigned int,PixelType,PixelType,PixelType,unsigned int,bool,bool)' being compiled [D:\a\_temp\build\ANTS-build\Examples\l_sccan.vcxproj]
            with
            [
                PixelType=double
            ]
D:\a\ANTs\ANTs\Examples\sccan.cxx(425,1): error C3861: 'random_shuffle': identifier not found [D:\a\_temp\build\ANTS-build\Examples\l_sccan.vcxproj] [D:\a\_temp\build\ANTS.vcxproj]
    Building Custom Rule D:/a/ANTs/ANTs/Examples/CMakeLists.txt
    simpleSynRegistration.cxx
    l_simpleSynRegistration.vcxproj -> D:\a\_temp\build\ANTS-build\Examples\Release\l_simpleSynRegistration.lib
    Building Custom Rule D:/a/ANTs/ANTs/Examples/CMakeLists.txt
    cli_simpleSynRegistration.cxx
    simpleSynRegistration.vcxproj -> D:\a\_temp\build\ANTS-build\Examples\Release\simpleSynRegistration.exe
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: Custom build for 'D:\a\_temp\build\CMakeFiles\39f0cb047635fed21b96ec6f4e938b35\ANTS-mkdir.rule;D:\a\_temp\build\CMakeFiles\39f0cb047635fed21b96ec6f4e938b35\ANTS-download.rule;D:\a\_temp\build\CMakeFiles\39f0cb047635fed21b96ec6f4e938b35\ANTS-update.rule;D:\a\_temp\build\CMakeFiles\39f0cb047635fed21b96ec6f4e938b35\ANTS-patch.rule;D:\a\_temp\build\CMakeFiles\39f0cb047635fed21b96ec6f4e938b35\ANTS-configure.rule;D:\a\_temp\build\CMakeFiles\39f0cb047635fed21b96ec6f4e938b35\ANTS-forcebuild.rule;D:\a\_temp\build\CMakeFiles\39f0cb047635fed21b96ec6f4e938b35\ANTS-build.rule;D:\a\_temp\build\CMakeFiles\39f0cb047635fed21b96ec6f4e938b35\ANTS-install.rule;D:\a\_temp\build\CMakeFiles\0588e425d3d6c95f2081dfaaf7c7525a\ANTS-complete.rule;D:\a\_temp\build\CMakeFiles\e50c365fbe07e4994beec7aae07962b2\ANTS.rule;D:\a\ANTs\ANTs\CMakeLists.txt' exited with code 1. [D:\a\_temp\build\ANTS.vcxproj]
Error: Process completed with exit code 1.
```

I don't know why this started happening, unless there's been some update to the runners that was outside our control.

I'll look into replacing the deprecated `random_shuffle` later

--- a/Examples/sccan.cxx
+++ b/Examples/sccan.cxx
@@ -419,11 +419,15 @@ PermuteMatrix(vnl_matrix<TComp> q, bool doperm = true)
   }
 // std::random_shuffle is deprecated since C++14,
 // see: http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2014/n4190.htm
-#if __cplusplus >= 201402L
-  std::shuffle(permvec.begin(), permvec.end(), std::random_device());
-#else
-  std::random_shuffle(permvec.begin(), permvec.end(), sccanRandom);
-#endif
+// 2023-08-30: MSVC 2019 on Github Actions runners having problems detecting C++ version,
+// so use new std::shuffle
+// #if __cplusplus >= 201402L
+//  std::shuffle(permvec.begin(), permvec.end(), std::random_device());
+// #else
+//   std::random_shuffle(permvec.begin(), permvec.end(), sccanRandom);
+// #endif
+std::shuffle(permvec.begin(), permvec.end(), std::random_device());
+
   //    for (unsigned long i=0; i < q.rows(); i++)
   //  // std::cout << " permv " << i << " is " << permvec[i] << std::endl;
   // for (unsigned long i=0; i < q.rows(); i++)

